tools/eggnog_mapper/eggnog_mapper.xml

```diff
@@ -0,0 +1,491 @@
+                <option value="verrNOG">Verrucomicrobiae  (verrNOG)</option>
+                <option value="virNOG">Viridiplantae  (virNOG)</option>
+            </param>
+            <param name="target_orthologs" type="select" label="target orthologs for functional transfer">
+                <option value="one2one">one2one</option>
+                <option value="many2one">many2one</option>
+                <option value="one2many">one2many</option>
+                <option value="many2many">many2many</option>
+                <option value="all" selected="true">all</option>
+            </param>
+            <param name="go_evidence" type="select"
+                    label="Select the set of GO terms that should be used for annotation">
+                <option value="experimental">experimental = Use only terms inferred from experimental evidence</option>
+                <option value="non-electronic" selected="true">non-electronic = Use only non-electronically curated terms</option>
+            </param>
+        </section>
+        <section name="seed_ortholog_options" expanded="false" title="Seed Ortholog Search Options">
+            <param name="seed_ortholog_evalue" type="float" value="0.001" label="Min E-value threshold">
+                <help>
+                Min E-value expected when searching for seed eggNOG ortholog. Applies to phmmer/diamond searches. 
+                Queries not having a significant seed orthologs will not be annotated.
+                </help>
+            </param>
+            <param name="seed_ortholog_score" type="integer" value="60" label="Minimum bit score threshold">
+                <help>
+                Min bit score expected when searching for seed eggNOG ortholog. 
+                Queries not having a significant seed orthologs will not be annotated.
+                </help>
+            </param>
+        </section>
+        <section name="output_options" expanded="false" title="Output Options">
+            <param name="no_file_comments" type="boolean" truevalue="--no_file_comments" falsevalue="" checked="true"
+                label="Exclude header lines and stats from output files"/>
+            <param name="no_annot" type="boolean" truevalue="--no_annot" falsevalue="" checked="false"
+                label="Skip functional annotation, reporting only hits"/>
+            <param name="report_orthologs" type="boolean" truevalue="--report_orthologs" falsevalue="" checked="false"
+                label="Output a file with the list of orthologs for each hits"/>
+        </section>
+    </inputs>
+    <outputs>
+        <data name="seed_orthologs" format="tabular" label="${tool.name} on ${on_string}: ${db.database}.seed_orthologs" from_work_dir="results.emapper.seed_orthologs">
+            <actions>
+                <action name="column_names" type="metadata" default="query_name,seed_eggNOG_ortholog,seed_ortholog_evalue,seed_ortholog_score"/>
```

Comment:
Oh, nice - we should make use more of this!!!

Reply:
that was all @jj-umn :)